run-tests.php

```diff
@@ -3167,14 +3169,14 @@ function get_summary(bool $show_ext_summary): string
     return $summary;
 }
 
-function show_start(int $start_time): void
+function show_start(int $start_timestamp): void
 {
-    echo "TIME START " . date('Y-m-d H:i:s', $start_time) . "\n=====================================================================\n";
+    echo "TIME START " . date('Y-m-d H:i:s', $start_timestamp) . "\n=====================================================================\n";
 }
 
-function show_end(int $end_time): void
+function show_end(int $start_timestamp, int|float $start_time, int|float $end_time): void
 {
-    echo "=====================================================================\nTIME END " . date('Y-m-d H:i:s', $end_time) . "\n";
+    echo "=====================================================================\nTIME END " . date('Y-m-d H:i:s', $start_timestamp + (int)(($end_time - $start_time)/1e9)) . "\n";
```

Comment:
This does not look correct. `$start_timestamp` is rounded down. The end time can still be `$start_time + 1`, even if `($end_time - $start_time) < 1`. I don't understand why this was changed at all.